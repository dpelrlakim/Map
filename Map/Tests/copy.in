newDefault m1
= m2 m1
empty m2
newInit m1
= m2 m1
size m2
erase m1 5
= m2 m1
= m3 m2
size m3
clear m1
= m2 m1
empty m2
