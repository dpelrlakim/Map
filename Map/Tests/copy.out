m1:
m1:
m2:
1
m1:
m2:
m1: (1,2) (3,4) (5,6) (7,8)
m2:
m1: (1,2) (3,4) (5,6) (7,8)
m2: (1,2) (3,4) (5,6) (7,8)
4
m1: (1,2) (3,4) (5,6) (7,8)
m2: (1,2) (3,4) (5,6) (7,8)
m1: (1,2) (3,4) (7,8)
m2: (1,2) (3,4) (5,6) (7,8)
m1: (1,2) (3,4) (7,8)
m2: (1,2) (3,4) (7,8)
m1: (1,2) (3,4) (7,8)
m2: (1,2) (3,4) (7,8)
m3: (1,2) (3,4) (7,8)
3
m1: (1,2) (3,4) (7,8)
m2: (1,2) (3,4) (7,8)
m3: (1,2) (3,4) (7,8)
m1:
m2: (1,2) (3,4) (7,8)
m3: (1,2) (3,4) (7,8)
m1:
m2:
m3: (1,2) (3,4) (7,8)
1
m1:
m2:
m3: (1,2) (3,4) (7,8)
